codegen/index.js

```diff
@@ -30,29 +31,26 @@ const types = {
   "selected": "Boolean",
   "size": "Number",
   "span": "Number",
-  "start": "Number",
-  "zoomAndPan": "String"
+  "start": "Number"
 };
+const reserved = ["module", "data", "type", "newtype", "class", "instance", "where", "derive", "if", "then", "else", "case", "of"];
```

Comment:
Since we're using these as record labels, there's no need to escape keywords (at least in recent versions of PureScript).

Reply:
They become function names though, like `div`.  The one that was giving me problems was `data`.  Apparently it's an HTML tag.

Reply:
I didn't realize `data` was a html tag. None of the others listed in `reserved` are html tags are they? @paf31 Was `data` not causing a problem in the previous version of this script?

Reply:
I assumed I got a newer version of `react-html-attributes` or something.  It seemed safer to add everything to the reserved list than have to remember and go back later if `type` or `module` become elements.